addons/dispatch/1.4.x/dispatch-10.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta2
+kind: ClusterAddon
+metadata:
+  name: dispatch
+  labels:
+    kubeaddons.mesosphere.io/name: dispatch
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.4.6"
+
+    # Dispatch
+    appversion.kubeaddons.mesosphere.io/dispatch: "1.4.6"
+    endpoint.kubeaddons.mesosphere.io/dispatch: /dispatch/dashboard 
+    docs.kubeaddons.mesosphere.io/dispatch: "https://docs.d2iq.com/dkp/dispatch/latest/"
+
+    # Tekton
+    appversion.kubeaddons.mesosphere.io/tekton: "0.14.2"
+    endpoint.kubeaddons.mesosphere.io/tekton: /dispatch/tekton/
+    docs.kubeaddons.mesosphere.io/tekton: "https://docs.d2iq.com/dkp/dispatch/latest/"
+
+    values.chart.helm.kubeaddons.mesosphere.io/dispatch: "https://raw.githubusercontent.com/mesosphere/charts/master/stable/dispatch/values.yaml"
+spec:
+  namespace: dispatch
+  kubernetes:
+    minSupportedVersion: v1.15.6
+  cloudProvider:
+    - name: aws
+      enabled: false
+    - name: azure
+      enabled: false
+    - name: gcp
+      enabled: false
+    - name: vsphere
+      enabled: false
+    - name: docker
+      enabled: false
+    - name: none
+      enabled: false
+  requires:
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: prometheus
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: cert-manager
+  chartReference:
+    chart: dispatch
+    repo: "https://mesosphere.github.io/dispatch"
+    version: "1.4.6"
+    values: |
+      ---
+      dispatch:
+        image: "mesosphere/dispatch:1.4.6"
+        testImage: "mesosphere/dispatch-helm-test:1.4.6"
+      kommander-ui:
+        image:
+          defaultTag: 6.98.0
```

Comment:
@TattdCodeMonkey is the correct version of UI (copied from #12)

Reply:
we should bump to `6.99.2` to match [kommander](https://github.com/mesosphere/charts/blob/master/stable/kommander/requirements.yaml#L19)